demo/index.html

```diff
@@ -22,6 +22,8 @@
       --iconWidth: 28px;
       --iconHeight: 28px;
       --activeButtonBg: #282828;
+      --closeMenuIconWidth: 18px;
```

Comment:
please use a generic padding for this.
or a max width for the submenu header with a text wrap around.